tests/unit/lib/Any_unittest.cc

```diff
@@ -693,10 +693,10 @@ TEST(Any, lazy_cloneable_visitable)
 	    any.clone_from(make_any<int>([=]() { return 111; })), std::logic_error);
 	EXPECT_TRUE(any.cloneable());
 	EXPECT_TRUE(any.visitable());
-	any = make_any<int>([=]() { return 222; });
-	EXPECT_FALSE(any.cloneable());
-	EXPECT_FALSE(any.visitable());
-	EXPECT_THROW(any.visit(nullptr), std::logic_error);
+//	any = make_any<int>([=]() { return 222; });
+//	EXPECT_FALSE(any.cloneable());
+//	EXPECT_FALSE(any.visitable());
+//	EXPECT_THROW(any.visit(nullptr), std::logic_error);
```

Comment:
@karlnapf I think now everything is visitable, except void functions

Reply:
i need to do some more refactoring here and rewrite the test, now this would be visitable and wouldn't throw